parse.go

```diff
@@ -0,0 +1,53 @@
+package vagrantutil
+
+import (
+	"bufio"
+	"strings"
+)
+
+// parseData parses the given vagrant type field from the machine readable
+// output (records).
+func (v *Vagrant) parseData(records [][]string, typeName string) (string, error) {
```

Comment:
This function doesn't need to be a part of `v` and can be a standalone function.